README.md

```diff
@@ -2,6 +2,12 @@
 
 ### Releases
 
+#### v8.4.1
```

Comment:
Why don't we use 8.4.0 version?

Reply:
You are right, there is no point to use 8.4.1 version. I will fix that.

Reply:
BTW, you didn't change version in package.json